app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java

```diff
@@ -249,11 +248,11 @@ public static void customTabsIntentEvent(final List<String> options) {
     }
 
     public static void downloadDialogDownloadEvent() {
-        TelemetryEvent.create(Category.ACTION, Method.DOWNLOAD, Object.DOWNLOAD_DIALOG, Value.DOWNLOAD).queue();
+        TelemetryEvent.create(Category.ACTION, Method.CLICK, Object.DOWNLOAD_DIALOG, Value.DOWNLOAD).queue();
```

Comment:
I'm okay with this change but I'd ask @bbinto to make sure we are not messing with her analytics.

Please also update the wiki after landing those probes:
https://github.com/mozilla-mobile/focus-android/wiki/Telemetry